README.md

```diff
@@ -0,0 +1,67 @@
+# building the current images
+
+**step 0: clone the repos**
+
+`git clone https://github.com/geoHeil/geodocker-geomesa.git`
```

Comment:
If we are merging this, it'd be good for the docs to point to the GeoDocker repo.